### PR TITLE
Refactor schemas

### PR DESCRIPTION
**Pull request by kmelve**

### CODE REVIEW

File: template/studio/schemas/author.js [hunk 1]
```diff
@@ -32,21 +29,14 @@ export default {
...
-      type: 'array',
-      of: [
-        {
-          title: 'Block',
-          type: 'block',
-          styles: [{title: 'Normal', value: 'normal'}],
-          lists: []
-        }
-      ]
+      type: 'simplePortableText',
```
Comment (mariuslundgard) [inline template/studio/schemas/author.js]: I think we should create unique portable text types for each place we use that. E.g. `bioPortableText` in this case.

Comment (kmelve) [reply template/studio/schemas/author.js]: Why?

File: template/studio/schemas/post.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        'This can be used to schedule post for publishing',
+    },
+    {
+      name: 'mainImage',
+      title: 'Main image',
+      type: 'mainImage',
+    },
+    {
+      name: 'excerpt',
+      type: 'text',
```
Comment (mariuslundgard) [inline template/studio/schemas/post.js]: This should probably be a portable text type, to allow for using code and emphasize text (but not links)?

File: template/studio/schemas/post.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+          type: 'reference',
+          to: {
+            type: 'category'
+          }
+        }
+      ]
+    },
+    {
+      name: 'body',
+      type: 'portableText',
```
Comment (mariuslundgard) [inline template/studio/schemas/post.js]: Would name this `postBodyPortableText`

File: template/studio/schemas/post.js (hunk 2 above)
Comment (kmelve) [reply template/studio/schemas/post.js]: The excerpt is used in post listings and summaries, and on Google? Does it makes sense to have links and stuff in that?

File: template/studio/schemas/types/mainImage.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+export default {
+  name: 'mainImage',
```
Comment (mariuslundgard) [inline template/studio/schemas/types/mainImage.js]: Name this `postMainImage`?

File: template/studio/schemas/types/portableText.js [hunk 5]
```diff
@@ -10,7 +10,7 @@
  */
 export default {
   type: 'array',
-  name: 'post.content',
+  name: 'portableText',
```
Comment (mariuslundgard) [inline template/studio/schemas/types/portableText.js]: Name this `postBodyPortableText`?

File: template/studio/schemas/types/simplePortableText.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+export default {
+  title: 'Portable Text',
+  name: 'simplePortableText',
```
Comment (mariuslundgard) [inline template/studio/schemas/types/simplePortableText.js]: `authorBioPortableText`

File: template/studio/schemas/post.js (hunk 2 above)
Comment (mariuslundgard) [reply template/studio/schemas/post.js]: It's used in the list of posts on the front page as well. Should not have links, but bold, italics, code, etc.

**Pull request merged**